content/en/blog/2020/on-demand-egress-gateways/index.md

```diff
@@ -0,0 +1,500 @@
+---
+title: On Demand Egress Gateways
+subtitle: Deploy Egress Gateways independently in a seamless way
+description: Deploying Egress Gateways independently to have fine-grained control of egress communication.
+publishdate: 2020-11-07
+attribution: Antonio Berben (At Deutsche Telekom - PAN-NET)
+keywords: [configuration,egress,gateway,external,service]
+target_release: 1.8.0
+---
+
+At [`Deutsche Telekom::PAN-NET`](https://pan-net.cloud/aboutus), we have embraced `Istio` as the umbrella to cover our services.
```

Comment:
```suggestion
At [Deutsche Telekom Pan-Net](https://pan-net.cloud/aboutus), we have embraced Istio as the umbrella to cover our services.
```

You've used ` ``foo`` ` throughout, which I am going to remove a lot of.

Reply:
Interesting. I swear It was part of the guidelines, at least in 1.7. But I don't find it anymore :man_shrugging:  . That was the reason I added the `back-quote`. Sorry for the confusion